<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecipeWidget</class>
 <widget class="QWidget" name="RecipeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>271</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="Layout_table"/>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="Layout_bottom"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>131</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="Layout_btns"/>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Layout_head"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
